MicrosoftQuickAuth.podspec

```diff
@@ -0,0 +1,29 @@
+Pod::Spec.new do |s|
+  s.name         = "MicrosoftQuickAuth"
+  s.version      = "0.0.5"
```

Comment:
How did we decide on 0.0.5?

Reply:
I used 0.0.1 ~ 0.0.4 for testing, the final version for our first release depends on us.

Reply:
As discussed on the meeting, we will start the version from 0.0.5